src/resources/SecurityCache/SecurityCache.ts

```diff
@@ -34,6 +35,13 @@ export default class SecurityCache extends Ressource {
         );
     }
 
+    listFilteredEntities(filters: SecurityCacheFilters) {
```

Comment:
The `filters` parameter should be optional because we could call the list without filter

Reply:
If you send an empty filter, API will return identities from all the providers in the org. Since we don't yet support that, we'll need to provide `providerId` in the filter for the call, although the call is capable of returning all identities. That's the reason we removed `providerId` from the method param thinking that the filter is going to contain it anyway.